articles/connectors/connectors-native-http.md

```diff
@@ -154,6 +154,9 @@ For example, if you're working in Visual Studio Code, follow these steps:
       }
    }
    ```
+Note: To find the thumbprint - from the left navigation of your app, select TLS/SSL settings, then select Private Key Certificates (.pfx) or Public Key Certificates (.cer). Find the certificate you want to use and copy the thumbprint.
```

Comment:
Edits to align and format the note correctly. We also need the source for the image to properly include, so let's just point at the doc that has the steps in case those steps change.

```suggestion

   > [!NOTE]
   >
   > To find the thumbprint, follow these steps:
   >
   > 1. On your logic app resource menu, under **Settings**, select **TLS/SSL settings** > **Private Key Certificates (.pfx)** or **Public Key Certificates (.cer)**.
   > 
   > 2. Find the certificate that you want to use, and copy the thumbprint.
   > 
   > For more information, review [Find the thumbprint - Azure App Service](../app-service/configure-ssl-certificate-in-code.md#find-the-thumbprint).
```